@here/harp-vectortile-datasource/lib/IGeometryProcessor.ts

```diff
@@ -56,11 +56,6 @@ export interface ILineGeometry {
  * line and polygon geometries, see [[VectorTileDecoder]].
  */
 export interface IGeometryProcessor {
-    /**
-     * The offset to apply the storage level of a tile to compute the zoom level.
-     */
-    storageLevelOffset?: number;
```

Comment:
should this be first deprecated?

Reply:
This wasn't part of the API, it's not exported in the package index.ts. Now it's only exposed indirectly because it's a parameter type of DataAdapter interface.

Reply:
Good point, I should also check the index.ts...